vm/errors.go

```diff
@@ -176,7 +176,7 @@ var ErrCouldNotCreateNewTokenIdentifier = errors.New("token identifier could not
 // ErrBLSPublicKeyAlreadyJailed signals that bls public key was already jailed
 var ErrBLSPublicKeyAlreadyJailed = errors.New("bls public key already jailed")
 
-// ErrInvalidEndOfEpochAccessAddress signals that end of epoch access address is invalid
+// ErrInvalidEndOfEpochAccessAddress signals that end of epoch access address is invalidgetExistingToken
```

Comment:
Delete getExistingToken from the end of the comment

Reply:
done